Dockerfile

```diff
@@ -1,11 +1,35 @@
+ARG HOST_UID=1000
```

Comment:
This seams to be part of the docker stuff PR ( #203  ).
Will you rebase this before merging?

Reply:
Yes, see the PR description above. I'll do that now :)